### DISCUSSION TITLE
Mac M1 芯片 使用认证工具，提示文件损坏

### DISCUSSION THREAD

**Discussion by thahao** [natfrp/wiki#448] [使用问题] [answered]
<img width="535" alt="image" src="https://github.com/natfrp/wiki/assets/63403724/f9700b3d-b54a-4f3b-9c72-22cd8afffcc5">

<img width="296" alt="image" src="https://github.com/natfrp/wiki/assets/63403724/07e4ebd5-a33f-4409-8b65-06a32afdf28d">

**Comment by fakeboboliu** [comment, marked as answer]
https://doc.natfrp.com/offtopic/auth-guest.html#macos-arm-note
